extensions/omni_httpc/omni_httpc.c

```diff
@@ -50,6 +50,12 @@ static h2o_multithread_receiver_t getaddr_receiver;
 
 static h2o_httpclient_connection_pool_t *connpool;
 static h2o_socketpool_t *sockpool;
+static h2o_httpclient_head_cb on_connect(h2o_httpclient_t *client, const char *errstr,
```

Comment:
needed a forward declaration, so I could refer to `on_connect` before its definition.